packages/tyxml-lwd/tyxml-lwd.0.1/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "Hello"
+description: "Make reactive webpages in Js_of_ocaml using Tyxml and Lwd"
+maintainer: ["fred@tarides.com"]
+authors: ["Frédéric Bour"]
+license: "MIT"
+homepage: "https://github.com/let-def/lwd"
+doc: "https://let-def.github.io/lwd/doc"
+bug-reports: "https://github.com/let-def/lwd/issues"
+depends: ["dune" "lwd" "tyxml" "js_of_ocaml" "js_of_ocaml-ppx"]
```

Comment:
could there be some constraints here (dune needs one to be a bit more explicit and lwd probably needs one if you don't want to be annoyed in future releases)